source/install/transport-encryption/config-mattermost.rst

```diff
@@ -59,14 +59,14 @@ Once complete, open the file ``config.json`` and modify the values ``ConnectionS
 .. codeblock:: none
 
   {
```

Comment:
I'm not 100% sure how to fix it but currently these are broken: https://docs.mattermost.com/install/transport-encryption/config-mattermost.html.

Reply:
@justinegeffen @DSchalla Do either one of you know if this PR fixes the missing code blocks? I've tried different things and all options look fine on Github preview, but I'm not sure about the actual documentation page.

![before](https://user-images.githubusercontent.com/29708087/70101744-e91e2000-1603-11ea-9b78-9febfdd13105.png)

Reply:
I've taken a look at it and as you said, it all looks correct. @DSchalla, were the examples by any chance copied and and pasted into the file? 

Reason I ask is I have a similar issue on some docs I am writing and I suspect it's because I copied and pasted, which somehow affected the structure. It all looks fine in the file but when I do a build, I have the same problem. I actually worked on a similar issue to this a while back - I'll take a look and see if I can find it and how I fixed it.

Reply:
Hm, I did indeed copy it, but it was from a plaintext JSON file, so no special characters or anything else. I'll take a look tomorrow as well!